index.html

```diff
@@ -248,13 +254,14 @@ <h4 class="title">Skills</h4>
           </ul>
         </div>
       </div>
-      </section>
+    </div>
   </section>
   <!--Contact Form Section-->
-  <section class="form-text">
+  <section class="wrapper">
```

Comment:
Again, the background image for this section is breaking. Kindly fix this also.
![image](https://user-images.githubusercontent.com/33268927/173057691-d9f5317a-65c0-4b80-be21-a4ad5a229315.png)

Reply:
- [ ] As mentioned, kindly add background image to match your design like Figma design. you can refer to [this lesson](https://developer.mozilla.org/en-US/docs/Web/CSS/background-image)

- [ ] [OPTIONAL] It will be better, if try to fix this extra space in right which make scroll in horizontal axis
- your design 
![image](https://user-images.githubusercontent.com/53402600/173138709-83b059a6-85ef-4cf8-8291-4915c3f0d55c.png)

Reply:
- [ ] **As mentioned by the previous reviewers,** kindly make sure to add the background image to match your design like Figma's design. For more information about adding a background image, you can refer to [this lesson](https://developer.mozilla.org/en-US/docs/Web/CSS/background-image) and improve your styling to match the one from Figma design as fon now, the whole page shows the **half** in a medium screen
### Figma design
![cf_3s](https://user-images.githubusercontent.com/83042742/173162490-038a4be0-b000-4416-8381-cba19c17e3eb.png)
### Your design
![bt](https://user-images.githubusercontent.com/83042742/173162390-1ceea5d9-a9b6-46ce-9b23-91b2dc00b823.png)